backend/curriculum_tracking/serializers.py

```diff
@@ -611,3 +611,20 @@ def get_oldest_card_in_review_time(self, instance):
 
     def get_total_cards_in_review(self, instance):
         return self._get_review_agile_cards(instance).count()
+
+
+class ReviewTrustSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.ReviewTrust
+        fields = [
+            "id",
+            "content_item",
```

Comment:
is this the content item or the content item title?  It's a little misleading.

It might be good to call a title a title.

Usually foreign key relationships are exposed as integers.

This would probably be the best version of the code:

```
fields = [
            "id",
            "content_item", # the default behaviour here is nice and consistent with the rest of the application.
            "content_item_title", # this is the title of a content item - see how descriptive the name is?
             "flavours",
            "user",
]

...

    content_item_title = serializers.SerializerMethodField('get_content_item_title')


```